pkg/api/rest/client.go

```diff
@@ -0,0 +1,69 @@
+package rest
+
+import (
+	"net/url"
+	"strings"
+
+	"github.com/pingcap/ticdc/pkg/httputil"
+)
+
+// RESTInterface includes a set of operations to interact with cdc REST apis
+type RESTInterface interface {
+	Verb(verb string) *Request
+	Post() *Request
+	Put() *Request
+	Get() *Request
+	Delete() *Request
+}
+
+type RESTClient struct {
+	// base is the root URL for all invocations of the client.
+	base *url.URL
+
+	// versionedAPIPath is a http url prefix with api version. eg. /api/v1.
+	versionedAPIPath string
```

Comment:
Would it be more concise if this is just a constant?

Reply:
When other versions of API are added to the code base in the future, it will be easy to do adapting work.